accname/index.html

```diff
@@ -610,7 +610,7 @@ <h4>Computation steps</h4>
                 <li id="comp_host_language_label">
                   <span id="step2E"><!-- Don't link to this legacy numbered ID. --></span><em>Host Language Label:</em> Otherwise, if the <code>current node</code>'s native markup provides an
                   [=attribute=] (e.g. <code>alt</code>) or [=element=] (e.g. HTML <code>label</code> or SVG <code>title</code>) that defines a text alternative, return that alternative in the form of
-                  a <code>flat string</code> as defined by the host language, unless the element is marked as presentational (<code>role="presentation"</code> or <code>role="none"</code>).
+                  a <code>flat string</code> as defined by the host language, unless the <code>current node</code> is exposed as presentational (<code>role="presentation"</code> or <code>role="none"</code>).
```

Comment:
Thinking this through (mostly for myself TBH):

- "marked" is explicit author intent
- "exposed" is what browsers decide to do (and is irrespective of author's intent)

I want to make sure that we're all really clear about what changing this wording means.

Reply:
what about "marked by the author and exposed by UA..."

Reply:
imo, simpler is better.  intent vs not is probably a topic better clarified outside of an algorithm step.  as then there can be nuances about whether that intent was even valid, hence why an author could mark something as presentational, but the browser is like "nope. not on that element you're not"

Reply:
+1 to keeping it "exposed as presentational" -- in the accname steps, I think sticking to talking about computed a11y info for simplicity makes the most sense. This feels like the wrong place to build in authoring-focused information.